languages/zh_CN.json

```diff
@@ -1955,71 +1955,581 @@
   "TXT_CODE_9d1d244f": "程序启动失败，输入输出流不可读：{{pipeName}}",
   "TXT_CODE_ca030197": "实例有其他任务正在进行中，无法进行此操作！",
   "TXT_CODE_aae2918f": "实例进程启动失败，请检查启动命令和运行环境等配置！",
-  "TXT_CODE_2093cc1a": "购买历史查询",
```

Comment:
You deleted the existing Chinese translation.

You need to undo the changes to the zh_CN.json file, you only need to change en_US.json.

Reply:
Do I just add the highlighted one here or do I reset it?

Reply:
Reset it

Reply:
I did. Or the Pufferfish changes too?